src/osha/oira/ploneintranet/z3cform/__init__.py

```diff
@@ -0,0 +1,9 @@
+_applied = False
+
+if not _applied:
+    from z3c.form.form import Form
+
+    # Zap the totally useless 'There were some errors' message.
+    Form.formErrorsMessage = None
+    del Form
+    _applied = True
```

Comment:
I was a bit puzzled about this and found out that it comes from copying the NuPlone folder.

https://github.com/euphorie/NuPlone/blob/05dd2949f0bcdc004ab2a3d4d0cc00bcd0f6675b/plonetheme/nuplone/z3cform/__init__.py#L6

I think wee can remove it:
```suggestion
```